AmplifyTestCommon/Helpers/AuthSignInHelper.swift

```diff
@@ -11,12 +11,12 @@ import XCTest
 public typealias CompletionType = (Bool, AuthError?) -> Void
 
 public struct AuthSignInHelper {
```

Comment:
this file can be reverted.

Reply:
these changes keep popping up because of SwiftFormat :)